pkg/index/index_inner_test.go

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright 2011 The Perkeep Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package index
+
+import (
+	"testing"
+	"time"
+
+	"perkeep.org/pkg/blob"
+	"perkeep.org/pkg/types/camtypes"
+)
+
+var testKvClaims = map[[2]string]camtypes.Claim{
```

Comment:
what are `[2]string`? just make a named type for that where you can document what [0] and [1] are?

Reply:
or make it a struct of two strings so they can be named